url-updater/src/main/java/com/devonfw/tools/ide/url/updater/AbstractUrlUpdater.java

```diff
@@ -190,58 +201,59 @@ protected boolean isSuccess(HttpResponse<?> response) {
   }
 
   /**
-   * Checks the download URL and takes the provided checksum into account instead of downloading the file and generating
-   * the checksum from it
+   * Checks if the download file checksum is still valid
    *
-   * @param url the URL of the download to check.
-   * @param urlVersion the {@link UrlVersion} where to store the collected information like status and checksum.
+   * @param url String of the URL to check
+   * @param urlVersion String of the version
```

Comment:
this change is incorrect and the old Javadoc of `urlVersion` was correct and more appropriate. Was this on accident or on purpose?